ui/main/src/app/modules/useractionlogs/useractionlogs.component.html

```diff
@@ -87,7 +87,7 @@
                 </table>
             </div>
             <div class="opfab-pagination" style="padding-bottom: 5px; margin-top: 5px">
-                <div style="margin-top: 12px">
+                <div style="margin-top: 12px;white-space: nowrap">
```

Comment:
This is not code introduced by my PR.
But anyway what you propose is wrong because with your suggestion, the display is not good. It is not vertically aligned anymore.